src/stringIndices.js

```diff
@@ -0,0 +1,12 @@
+function getIndex() {
+  let phrase          = document.getElementById('phrase').value
+  let indexNum        = document.getElementById('indexNum').value
+
```

Comment:
So I was told that formatting like this forces other developers in line to follow the same format and that is why its frowned upon. So in a project of your own I don't see any issue with it, and to be honest I'd prefer it myself as well, but I could see how in a bigger team environment this could get you some flak.